### ISSUE TITLE
maps

### ISSUE THREAD

**Issue by jeffnv** [MisterDeejay/MosaApp#2]
when is the google maps phase? Phase 1 I am guessing. This can be tricky, but is the highest priority, much higher than comments on anything.

**Comment by rglassett** [comment]
I agree. You'll want Backbone views for the map, search parameters, and search results. Price, review score, and distance would be good filters.

**Comment by MisterDeejay** [comment]
Thanks for the feedback guys! I will definitely work on adding more detail in my wireframes & timeline today. I also really like your suggestions for different filters.
